elibrary_parser/types.py

```diff
@@ -2,6 +2,21 @@
 
 
 class Publication:
+    """ Gets rid of unnecessary and
+    Finds similarities between given authors
+
+     Attributes
+     ----------------------
+     title: str
+        publication title
+     authors: str
+        publication authors
+     info: str
+        publication info (journal, etc.)
+     link: str
+        link for the publication
+    """
+
     def __init__(self, title, authors, info, link):
```

Comment:
Ещё вот здесь хорошо бы типы пометить
```suggestion
    def __init__(self, title: str, authors: str, info: str, link: str):
```